docs/deploying-kas-fleet-manager-to-openshift.md

```diff
@@ -147,6 +167,27 @@ make deploy/service IMAGE_TAG=<your-image-tag-here> <OPTIONAL_PARAMETERS>
 - `STRIMZI_OPERATOR_ADDON_ID`: The id of the Strimzi operator addon. Defaults to `managed-kafka-qe`.
 - `KAS_FLEETSHARD_ADDON_ID`: The id of the kas-fleetshard operator addon. Defaults to `kas-fleetshard-operator-qe`.
 
+### Using an Image from a Private External Registry
+If you are using a private external registry, a docker pull secret must be created in the namespace where KAS Fleet Manager is deployed and linked to the service account that KAS Fleet Manager uses.
+
+Create a docker pull secret with credentials that has access to pull the KAS Fleet Manager image from the private external registry.
+```
+oc create secret generic kas-fleet-manager-pull-secret \
+  --from-file=.dockerconfigjson=<path-to-docker-config-json> \
+  --type=kubernetes.io/dockerconfigjson
+```
+
+Link the pull secret to the KAS Fleet Manager service account
+```
+oc secrets link kas-fleet-manager kas-fleet-manager-pull-secret --for=pull
+```
+
+Delete the KAS Fleet Manager pod to restart the deployment
```

Comment:
Is there a chance there could be more than one instance of KFM Pods? just checking the language for plurality.

Reply:
also this can be removed, if we create the secret, service account before making the KFM deployment..

Reply:
yes there can be multiple, i can change that line to:
```suggestion
Delete the KAS Fleet Manager pod(s) to restart the deployment
```

wdyt?

Reply:
this looks good.

Reply:
i think i dont have permissions to accept the change, can you make the changes @JameelB ? it looks good to me

Reply:
just changed it there, thanks! :)